exec/Aula.hs

```diff
@@ -8,12 +8,14 @@ import Text.Show.Pretty
 
 import Frontend
 import Config
+import Action.Smtp
 
 
 main :: IO ()
 main = do
     setCurrentDirectoryToAulaRoot
     cfg <- readConfig CrashMissing
+    checkSendMail cfg
```

Comment:
Hum do we want to try send a test email to user@example.com at each run of the server?